syntax = "proto3";
package dfi;


message DFIAllocMultipleSegmentsRequest {
    repeated DFIAllocSegmentsRequest allocSegmentRequests = 1;
}

message DFIAllocSegmentsRequest {
    string name = 1;
	uint32 segments_count = 2;
	uint64 segments_size = 3; //with footer size (full segment size)
	uint32 buffer_type = 4;
	bool cache_align = 5;
}

